include/veriblock/blockchain/blocktree.hpp

```diff
@@ -51,7 +51,7 @@ struct BlockTree {
       return state.addStackFunction("bootstrap()");
     }
 
-    if (!block_index_.empty() && !getBlockIndex(block.getHash())) {
+    if (block_index_.empty() && !getBlockIndex(block.getHash())) {
```

Comment:
No sense checking getBlockIndex if block_index_ is empty.

Reply:
fixed